pynicotine/plugins/now_playing_sender/__init__.py

```diff
@@ -151,4 +163,30 @@ def song_change(self, _connection, _sender_name, _object_path, _interface_name,
         # Keep track of which song is playing
         self.last_song_url = changed_song_url
 
-        self.send_now_playing()
+        self.send_now_playing(broadcast=True)
+
+    def send_now_playing(self, broadcast=False, local=False):
+        """ Broadcast Now Playing in selected rooms, or Send in current chat """
+
+        playing = self.core.now_playing.get_np()
+
+        if local or not playing:
+            # Display output locally
+            self.echo_message(playing)
+            return
+
+        if broadcast:
+            # Broadcast in selected rooms
+            for room in self.settings["rooms"]:
+                self.send_public(room, playing)
+        else:
+            # Send in current chat only
+            self.send_message(playing)
+
+    def now_playing_command(self, args, **_unused):
+        """ /now command """
+
+        self.send_now_playing(
+            broadcast=(args == "broadcast"),
+            local=(not args)
```

Comment:
The purpose of this command for 20 years has been to broadcast what you're listening to. I'm not willing to make it local by default, otherwise the whole feature might as well be removed.

Reply:
Good point. The default action should be send (to current chat only).